### DISCUSSION TITLE
Retrying a beatmap in editor causes crash to desktop (Direct3D11 renderer)

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22889] [Q&A] [answered]
Version: current master with [this framework](https://github.com/peppy/osu-framework/tree/renderer-config-fallback)

https://user-images.githubusercontent.com/86934170/226000329-006783c1-c4f2-48e9-8ca8-87d96a359901.mp4

[performance.log](https://github.com/ppy/osu/files/11005148/performance.log)
[database.log](https://github.com/ppy/osu/files/11005149/database.log)
[network.log](https://github.com/ppy/osu/files/11005150/network.log)
[runtime.log](https://github.com/ppy/osu/files/11005151/runtime.log)

Another case where the game has crashed on multi spectator then I had to restart my PC because of the black screen (any binds like Ctrl+Alt+Del stopped working)

[runtime.log](https://github.com/ppy/osu/files/11008030/runtime.log)
[network.log](https://github.com/ppy/osu/files/11008031/network.log)
[database.log](https://github.com/ppy/osu/files/11008032/database.log)
[performance.log](https://github.com/ppy/osu/files/11008033/performance.log)

**Comment by peppy** [comment]
This is going to be related to blur shader issues cc/ @smoogipoo  (pretty sure you're aware)

**Comment by smoogipoo** [comment, marked as answer]
The log suggests an out of memory exception. I'll try to reproduce but it's possible that it's caused by https://github.com/ppy/osu-framework/pull/5684 which I don't believe is in the framework version which osu-master is on.

**Reply by Loreos7** [reply]
Tested again, now its fine.
